docs/versioned_docs/version-2.50.0-LTS/widgets/date-range-picker.md

```diff
@@ -4,37 +4,35 @@ title: Date-range Picker
 ---
 # Date-Range Picker
 
-The **Date-Range Picker** widget allows users to select a range of dates.
+The **Date-Range Picker** component allows users to select a range of dates.
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Properties
 
 | <div style={{ width:"135px"}}> Property  </div>    | <div style={{ width:"100px"}}> Description </div> |
 |:----------- |:----------- |
-| Default start date | Set the start date to be selected by default in the widget |
-| Default end date | Set the start date to be selected by default in the widget |
+| Default start date | Set the start date to be selected by default in the component. |
+| Default end date | Set the start date to be selected by default in the component. |
 | Format | The format of the date selected by the date picker. Default date format is **DD/MM/YYYY**. Date format should be followed as ISO 8601 as mentioned in the [moment documentation](https://momentjs.com/docs/). |
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Events
 
-Date range picker supports the following events:
-
 | <div style={{ width:"100px"}}> Event  </div>    | <div style={{ width:"100px"}}> Description </div> |
 |:----------- |:----------- |
-| On select | The On select event is triggered when the a start date and end date is selected on the picker. Just like any other event on ToolJet, you can set multiple handlers for on select event. |
+| On select | Triggered when a start date and end date are selected. You can set multiple handlers for this event. |
 
 :::info
 Check [Action Reference](/docs/category/actions-reference) docs to get the detailed information about all the **Actions**.
 :::
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Component Specific Actions (CSA)
 
```

Comment:
Hey @rudrakshnalbalwar, here on line no. 43 you need to remove the paddingBottom